cmd/download-plugins/main.go

```diff
@@ -108,6 +117,9 @@ func run() error {
 				continue
 			}
 		}
+		if releaseSince != "" && pluginRelease.ReleaseTag < releaseSince {
```

Comment:
Can use string comparison here as release tags are in the format `yyyyMMdd.<num>`.